### PR TITLE
[IMP] field service: default warehouse

### PR DESCRIPTION
**Pull request by zulu-odoo**
task id: 2948598

### CODE REVIEW

**Review by toaa-odoo — CHANGES_REQUESTED** [review]

**Review by toaa-odoo — APPROVED** [review]

**Comment by xpl-odoo** [comment]
Since you assigned yourself as a reviewer on the task @jcs-odoo

File: content/applications/services/fsm/default_warehouse.rst [hunk 1]
```diff
@@ -3,10 +3,11 @@ User default warehouse
 ======================
 
 Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
-van or those who always resupply from the same warehouse.
+van or those who always resupply from the same warehouse. It also allows field workers to switch
+between warehouses from their profile.
```
Comment (jcs-odoo) [inline content/applications/services/fsm/default_warehouse.rst:7]: It seems this is the main point you want to add in this PR, yet, you don't explain how to do it. Please add an explanation, as the current one is only accessible as an admin, and not as a regular user.

File: content/applications/services/fsm/default_warehouse.rst [hunk 2]
```diff
@@ -27,16 +27,22 @@ User account
...
 To set up a default warehouse for a specific user, go to :menuselection:`Settings --> Users -->
-Manage users`, open a user, then go to the :guilabel:`Preferences` tab, scroll down to
+Manage users`, open a user, then go to the :guilabel:`Preferences` tab. Scroll down to
 :guilabel:`Inventory`, and select the default warehouse from the drop-down menu.
 
 .. image:: default_warehouse/user-default.png
    :alt: Selection of a default warehouse on a user profile.
 
+.. tip::
+   You can follow the same steps to easily change your default warehouse as needed.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/services/fsm/default_warehouse.rst:38]: @zulu-odoo you marked this as resolved but didn't address it nor changed it?

Also, I think this tip isn't really necessary. Isn't this the case with practically all settings?

File: content/applications/services/fsm/default_warehouse.rst [hunk 3]
```diff
@@ -26,17 +27,23 @@ than one warehouse in your database.
 User account
 ------------
 
-To set up a default warehouse for a specific user, go to :menuselection:`Settings --> Users -->
-Manage users`, open a user, then go to the :guilabel:`Preferences` tab, scroll down to
-:guilabel:`Inventory`, and select the default warehouse from the drop-down menu.
+To set up a default warehouse for a specific user, open the **Settings** application, then go to
+:menuselection:`Users --> Manage users`, select a user, then go to the :guilabel:`Preferences` tab.
```
Comment (jcs-odoo) [inline content/applications/services/fsm/default_warehouse.rst:31]: I think the current content is better
`go to :menuselection:`Settings --> Users -->`
(I know you initially left it like that :) )

File: content/applications/services/fsm/default_warehouse.rst (hunk 1 above)
Comment (zulu-odoo) [reply content/applications/services/fsm/default_warehouse.rst:7]: Indeed, thank you! I added it as a tip (.. tip::
   To switch your own default warehouse from anywhere in the database, click your **profile
   icon** in the upper right corner of the screen. Then, go to :menuselection:`My Profile -->
   Preferences --> Default Warehouse`.) Let me know if it makes sense to you.

Comment (zulu-odoo) [reply content/applications/services/fsm/default_warehouse.rst:7]: Updated to two headings as discussed

File: content/applications/services/fsm/default_warehouse.rst (hunk 3 above)
Comment (zulu-odoo) [reply content/applications/services/fsm/default_warehouse.rst:31]: I reverted it back to how it was, to me both options are fine. With the "go to (:menuselection:Settings --> Users -->`)" version, it wasn't super clear during the 1st review which "settings" we were talking about. That's why we added "Settings application" version. Lmk if it changes anything or you :)

File: content/applications/services/fsm/default_warehouse.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (zulu-odoo) [reply content/applications/services/fsm/default_warehouse.rst:38]: Indeed, during the in-person review, I was kindly given the blessing to keep the "easily" this time ;) Anyway, I removed the whole tip now :)

**Comment by jcs-odoo** [comment]
@robodoo r+

**Comment by jcs-odoo** [comment]
@robodoo retry

**Pull request closed without merging**